app/controllers/competencies_controller.rb

```diff
@@ -12,14 +13,7 @@ def index
   # GET /competencies/1
   # GET /competencies/1.json
   def show
-    respond_with @competency do |format|
-      format.json do
-        render json: {}, status: :ok
-      end
-      format.html do
-        redirect_to @competency
-      end
-    end
+    render json: Competency.find(params[:id])
```

Comment:
Right, I forgot about the private set_competency method... How about: 
`render json: @competency` then?